spec/utils_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'spec_helper'
+
+RSpec.describe Kitchen::Utils do
+  describe '.search_path_to_type(search_path' do
```

Comment:
did you want the `(search_path` at the end?

Reply:
whoops! no